workers/jobs/google_scraper.go

```diff
@@ -0,0 +1,92 @@
+package jobs
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/gocraft/work"
+	"github.com/gutakk/go-google-scraper/models"
+	"github.com/gutakk/go-google-scraper/services/google_scraping_service"
+)
+
+const (
+	MaxFails = 3
```

Comment:
How about making this a configuration environment variable? See the [12 Factor app](https://12factor.net/config) for reference.

Reply:
In this case our production app is deployed to Heroku. What is the way should we do about env variables on production?
1. Add ENV directly to Heroku config
2. Create `.env` and `.env.release` in local machine and let Dockerfile copy that 2 files when build

Reply:
The first option would be the Heroku way 👍 Nothing wrong with the second option but it's good to use nice features like env variables management provided by Heroku.

Also just to clarify:

```
Create .env and .env.release in local machine
```

`local machine` would be the CI server 🧐